src/particle.jl

```diff
@@ -56,6 +77,15 @@ function ptype_for_code(code::Integer)
     return PUnknown
 end
 
+
```

Comment:
this is the docstring for the method that takes a particle as argument. I would move it to the method defined lower in the file. The methods that take type arguments don't need individual documentation in my opinion